js/alerts.js

```diff
@@ -90,6 +90,122 @@ function loadSelectors() {
+                    for (var j = 0; j < options.length; j++) {
+                        value += "<option ";
+                        if (alert['status'] == options[j]) {
+                            value += "selected";
+                        }
+                        value += ">" + options[j] + "</option>";
+                    }
+                    value += "</select>";
+                    cell.onclick = "";
+                } else if (fields[i] == "investigate") {
+                    var investigate = '';
+                    var defaultValue = "<p><a href=\"#\" onClick=\"updateBug(" + alert['id'] + ", '" + alert['bug'] + "', '" + alert['status'] + "');\">add bug #</a></p>";
+                    if (nvl(alert['bug'], "") != "") {
+                        var bugz = alert['bug'].split(',');
+                        defaultValue = "";
+                        for (var b = 0; b < bugz.length; b++) {
+                            var bugid = bugz[b].trim();
+                            defaultValue += "<a href=\"https://bugzilla.mozilla.org/show_bug.cgi?id=" + bugid + "\">" + bugid + "</a>";
+                            defaultValue += "<a href=\"#\" onClick=\"updateBug(" + alert['id'] + ", '" + alert['bug'] + "', '" + alert['status'] + "');\"> + </a><br />";
+                        }
+                    }
+                    investigate += defaultValue;
+                    defaultValue = "<p><a href=\"#\" onClick=\"updateTbplURL(" + alert['id'] + ", '" + alert['tbplurl'] + "');\">add tbpl link</a></p>";
+                    if (alert['tbplurl'] != "") {
+                        if (!(alert['branch'].endsWith("Non-PGO")) && (alert['tbplurl'] != null)) {
+                            var url = alert['tbplurl'].split("talos");
+                            alert['tbplurl'] = url[0] + "pgo%20talos" + url[1];
+                            defaultValue = "<p><a href=\"" + alert['tbplurl'] + "\">tbplurl</a><p>";
+                        } else {
+                            defaultValue = "<p><a href=\"" + alert['tbplurl'] + "\">tbplurl</a></p>";
+                        }
+                        defaultValue = "<a href=\"" + alert['tbplurl'] + "\">tbplurl</a>";
+                    }
+                    investigate += defaultValue;
+                    value = investigate;
+                    cell.onclick = "";                
+                }
+                textNode = value || 0;
+                if (alert['backout'] !=null && fields[i] == "percent"){
+                    url = "http://alertmanager.allizom.org:8080/alerts.html?rev="+alert['backout'];
+                    textNode = "<p title=\"Backout:"+alert['backout']+"\"><a target=\"_blank\" href=\"" + url + "\">"+textNode + "**"+ "</a></p>";
+                }
+                cell.innerHTML = textNode; 
+            }
+            return 1
+        }
+
+    var results;
+        $(document).ready(function () {
```

Comment:
I am not sure if the $(document).ready() call should be in here or not.

we use js/alerts.js in these files:
alerts.html
bug_conflict.html
expired.html
win8.html (this could be removed - it isn't impportant, but the concept could be beneficial)